### PR TITLE
feat: convert to a Gatsby theme

### PR DESCRIPTION
**Pull request by maxpou**
## Content
* feat: convert to a Gatsby theme
* feat(Theme): use tokens (https://www.gatsbyjs.org/blog/2019-04-29-component-shadowing/#changing-styling)
* docs(README): convert yarn script to npm (https://twitter.com/housecor/status/1109177783689859074)
* refactor: use hooks and change siteConfig structure
BREAKING CHANGE: footerLinks section changed! See `siteConfig.js`.

## TODO
- [x] refactor: remove all siteConfig import (`import siteConfig from '../../data/siteConfig'`) => use hooks
- [x] doc: update readme + how to install.md (adapt footer breaking change)
- [x] fix: in themes, `gatsby-config.js` use  gatsby-starter-morning-dew's `siteConfig.js` data.
- [x] fix: 404 page should be accessible on theme mode 
- [x] fix: in siteConfig, `iconClassName ` is unused. Use it or delete it
- [ ] fix: Fix website icons & `gatsby-plugin-manifest`. Only happen when it's used as a theme.
- [x] fix: Fix cover + avatar bio image (not deployed)
- [x] feat: add token folder for colors  [OPTIONAL]
- [ ] feat: add token mediaQueries [OPTIONAL]

## Related

https://github.com/maxpou/maxpou.fr/pull/34

### CODE REVIEW

File: content/pages/how-to-install/how-to-install.md [hunk 1]
```diff
@@ -63,47 +83,56 @@ module.exports = {
...
+        },
+        {
+          label: 'Website',
+          url: 'https://www.maxpou.fr',
+          iconClassName: 'fa fa-globe',
+        },
+        {
+          label: 'Twitter',
+          url: 'https://twitter.com/_maxpou',
+          iconClassName: 'fa fa-twitter',
```
Comment (maxpou) [inline content/pages/how-to-install/how-to-install.md]:
```suggestion

```

- [x] remove `iconClassName`

**Comment by JeroenKnoops** [comment]
Do you want me to test some of my presentation with this change?

**Comment by maxpou** [comment]
@JeroenKnoops 

> Do you want me to test some of my presentation with this change?

Hmm it's not fully ready (yet). As you can see in the preview deploy: https://deploy-preview-55--gatsby-starter-morning-dew.netlify.com
I've issues with images: header + author avatar (I need to push the fix).

Once it's ready, I'd be happy to have you as a reviewer!! 😀

By the way, are you using this starter somewhere?

**Comment by JeroenKnoops** [comment]
> By the way, are you using this starter somewhere?

Yes. We have a coders blog: https://github.com/040code/040code.github.io
We've changed it a little bit, because we have multiple authors.

**Pull request merged**